C2CRIBuildDir/projects/C2C-RI/src/RI_Logging/src/org/fhwa/c2cri/logger/ActionLogAppender.java

```diff
@@ -92,217 +55,19 @@ public void removeListener(ActionLogAppenderListener actionListener){
         listener = null;
     }    
 
-    /**
-     * Derived appenders should override this method if option structure
-     * requires it.
-     */
-    public void activateOptions() {
-		// original implementation was empty
-    }
-
-    /**
-     * Add a filter to end of the filter list.
-     *
-     * @since 0.9.0
-     */
-    public void addFilter(Filter newFilter) {
-        if (headFilter == null) {
-            headFilter = tailFilter = newFilter;
-        } else {
-            tailFilter.setNext(newFilter);
-            tailFilter = newFilter;
-        }
-    }
-
-    /**
-     * Subclasses of <code>AppenderSkeleton</code> should implement this method
-     * to perform actual logging. See also {@link #doAppend
-     * AppenderSkeleton.doAppend} method.
-     *
-     * @since 0.9.0
-     */
-    public void append(LoggingEvent event){
-        if (this.listener != null){
-            if (event.getLevel().isGreaterOrEqual(Priority.INFO))
-                this.listener.appenderUpdate(event.timeStamp, event.getRenderedMessage());
-        }
-    }
-
-    /**
-     * Clear the filters chain.
-     *
-     * @since 0.9.0
-     */
-    public void clearFilters() {
-        headFilter = tailFilter = null;
-    }
 
-    /**
-     * Finalize this appender by calling the derived class'
-     * <code>close</code> method.
-     *
-     * @since 0.8.4
-     */
-    protected void finalize() {
-        // An appender might be closed then garbage collected. There is no
-        // point in closing twice.
-        if (this.closed) {
-            return;
+	@Override
+    public void append(LogEvent event){
```

Comment:
Can we add JavaDoc to describe this func?

Reply:
Add a Javadoc for the function